### PR TITLE
contrib: import system containers

### PR DESCRIPTION
**Pull request by runcom**
@giuseppe PTAL

### CODE REVIEW

**Comment by giuseppe** [comment]
LGTM!

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM, but Travis is having some whitespace issues.

**Comment by ashcrow** [comment]
LGTM

**Comment by runcom** [comment]
@mrunalp @rhatdan PTAL

**Comment by runcom** [comment]
@jwhonce ptal

File: contrib/system_containers/rhel/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM rhel7:7-released
```
Comment (jwhonce) [inline contrib/system_containers/rhel/Dockerfile]: Why were the OIT headers removed?  I assume OIT will still be responsible for the rhel builds.  Yes?

Comment (runcom) [reply contrib/system_containers/rhel/Dockerfile]: I can bring them in again, I removed them mainly cause it's RH specific

Comment (runcom) [reply contrib/system_containers/rhel/Dockerfile]: Put the headers back, I just removed references to internal repos and IRC channels but the #oit headeres are back now. Thanks Jhon.

**Comment by rhatdan** [comment]
This LGTM, but I will rely on @jwhonce and @giuseppe since they are subject matter experts.

**Comment by runcom** [comment]
Note to self: we'll need the same PR against supported branches (1.7, 1.8 as of today)

**Comment by runcom** [comment]
@jupierce could you take a look at the RH bits in here

File: contrib/system_containers/rhel/Dockerfile (hunk 1 above)
Comment (jupierce) [reply contrib/system_containers/rhel/Dockerfile]: The oit headers are only intended to be "seen" in distgit. They are added dynamically when moving the file from this upstream repo to the downstream distgit. You can make notes in this file documenting how things work for your own purposes, but the oit comments do not need to be preserved.

File: contrib/system_containers/rhel/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    if test -e /usr/libexec/cni; then cp -Lr /usr/libexec/cni/* /exports/hostfs/opt/cni/bin/; fi
+
+COPY manifest.json tmpfiles.template config.json.template service.template /exports/
+
+COPY set_mounts.sh /
+COPY run.sh /usr/bin/
+
+CMD ["/usr/bin/run.sh"]
+
+LABEL \
```
Comment (jupierce) [inline contrib/system_containers/rhel/Dockerfile]: oit will add all necessary build labels when migrating this file downstream. So feel free to remove anything OSBS/brew specific (com.redhat.component, name, release, version).

Comment (runcom) [reply contrib/system_containers/rhel/Dockerfile]: ok, I'm fine keeping or removing them :) @giuseppe @jwhonce wdyt?

Comment (giuseppe) [reply contrib/system_containers/rhel/Dockerfile]: I am fine either way.  Although I don't see the point of deliberarely making the Dockerfile different if we are not exposing anything that shouldn't be there.

**Comment by giuseppe** [comment]
anything blocking this?

**Comment by runcom** [comment]
> anything blocking this?

nope, I think we can actually merge this, @mrunalp PTAL

**Comment by mrunalp** [comment]
LGTM

**Comment by runcom** [comment]
we need this on other supported branches as well (/cc @jupierce ) - I'll make some PR in a moment

**Pull request merged**